.github/workflows/pip_rc_install.yml

```
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies and the latest RC of pysatNASA from test pypi.
+# This test should be manually run before a pysatMadrigal RC is officially approved and versioned.
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test install of latest RC from pip
+
+on: [push]
```

```suggestion
on: [workflow_dispatch]
```
change before final merge